c/include/libsbp/v4/sbp_msg.h

```diff
@@ -0,0 +1,2362 @@
+/*
+ * Copyright (C) 2015-2021 Swift Navigation Inc.
+ * Contact: https://support.swiftnav.com
+ *
+ * This source is subject to the license found in the file 'LICENSE' which must
+ * be be distributed together with this source. All other rights reserved.
+ *
+ * THIS CODE AND INFORMATION IS PROVIDED "AS IS" WITHOUT WARRANTY OF ANY KIND,
+ * EITHER EXPRESSED OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE IMPLIED
+ * WARRANTIES OF MERCHANTABILITY AND/OR FITNESS FOR A PARTICULAR PURPOSE.
+ */
+
+/*****************************************************************************
+ * Automatically generated
+ * with generate.py. Please do not hand edit!
+ *****************************************************************************/
+
+#ifndef LIBSBP_V4_SBP_MSG_H
+#define LIBSBP_V4_SBP_MSG_H
+
+#include <libsbp/v4/acquisition.h>
+#include <libsbp/v4/bootload.h>
+#include <libsbp/v4/ext_events.h>
+#include <libsbp/v4/file_io.h>
+#include <libsbp/v4/flash.h>
+#include <libsbp/v4/gnss.h>
+#include <libsbp/v4/imu.h>
+#include <libsbp/v4/linux.h>
+#include <libsbp/v4/logging.h>
+#include <libsbp/v4/mag.h>
+#include <libsbp/v4/navigation.h>
+#include <libsbp/v4/ndb.h>
+#include <libsbp/v4/observation.h>
+#include <libsbp/v4/orientation.h>
+#include <libsbp/v4/piksi.h>
+#include <libsbp/v4/sbas.h>
+#include <libsbp/v4/settings.h>
+#include <libsbp/v4/solution_meta.h>
+#include <libsbp/v4/ssr.h>
+#include <libsbp/v4/system.h>
+#include <libsbp/v4/tracking.h>
+#include <libsbp/v4/user.h>
+#include <libsbp/v4/vehicle.h>
+#include <stdbool.h>
+#include <stddef.h>
+#include <stdint.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/*
+ * Used to hold an individual sbp_msg struct so that the message can be used in
+ * all general purpose functions within this file.
+ */
+typedef union {
+  sbp_msg_acq_result_t acq_result;
+  sbp_msg_acq_result_dep_c_t acq_result_dep_c;
+  sbp_msg_acq_result_dep_b_t acq_result_dep_b;
+  sbp_msg_acq_result_dep_a_t acq_result_dep_a;
+  sbp_msg_acq_sv_profile_t acq_sv_profile;
+  sbp_msg_acq_sv_profile_dep_t acq_sv_profile_dep;
+  sbp_msg_bootloader_handshake_req_t bootloader_handshake_req;
+  sbp_msg_bootloader_handshake_resp_t bootloader_handshake_resp;
+  sbp_msg_bootloader_jump_to_app_t bootloader_jump_to_app;
+  sbp_msg_nap_device_dna_req_t nap_device_dna_req;
+  sbp_msg_nap_device_dna_resp_t nap_device_dna_resp;
+  sbp_msg_bootloader_handshake_dep_a_t bootloader_handshake_dep_a;
+  sbp_msg_ext_event_t ext_event;
+  sbp_msg_fileio_read_req_t fileio_read_req;
+  sbp_msg_fileio_read_resp_t fileio_read_resp;
+  sbp_msg_fileio_read_dir_req_t fileio_read_dir_req;
+  sbp_msg_fileio_read_dir_resp_t fileio_read_dir_resp;
+  sbp_msg_fileio_remove_t fileio_remove;
+  sbp_msg_fileio_write_req_t fileio_write_req;
+  sbp_msg_fileio_write_resp_t fileio_write_resp;
+  sbp_msg_fileio_config_req_t fileio_config_req;
+  sbp_msg_fileio_config_resp_t fileio_config_resp;
+  sbp_msg_flash_program_t flash_program;
+  sbp_msg_flash_done_t flash_done;
```

Comment:
would be nice if these could be sorted

Reply:
Hmm, they were previously. Not sure what went wrong here